keyboards/planck/keymaps/fabian/config.h

```diff
@@ -1,7 +1,7 @@
 #ifndef CONFIG_USER_H
```

Comment:
Could you replace this block: 
```h
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include "config_common.h"
```
with just `#pragma once`, and remove the `#endif` at the end?